apps/supernova/src/lib/createFiltersSlice.js

```diff
@@ -25,8 +25,26 @@ const parsePredefinedFilters = (predefinedFilters) => {
   return predefinedFilters
 }
 
+const parseInitialFilters = (initialFilters, filterLabels) => {
+  if (!initialFilters) return initialFiltersState.initialFilters
+
+  if (typeof initialFilters !== "object") {
+    console.warn("[supernova]::parseInitialFilters: initialFilters object is not an object")
+    return {}
+  }
+
+  // Check if all keys are in filterLabelValues
+  if (!Object.keys(initialFilters).every((key) => filterLabels.includes(key))) {
+    console.warn("[supernova]::parseInitialFilters: Some keys of the initialFilters object are not in the labels")
```

Comment:
Change message to:
"Some keys of the initialFilters object are not valid filter labels. They must be configured as filterLabels first."